packages/react-inbox/src/actions/message-count.ts

```diff
@@ -0,0 +1,12 @@
+export const QUERY_MESSAGE_COUNT = `
+  query MessageCount($isRead: Boolean!) {
+    messageCount(params: {
+      isRead: $isRead
+    })
+  }
+`;
+
+export const getMessageCount = async (client, { isRead = false }) => {
```

Comment:
do we have to pass client in?

Reply:
I believe so, I cannot think of another pattern without using hooks, or being on the window object